jarjar/src/main/java/com/eed3si9n/jarjar/util/IoUtil.java

```diff
@@ -53,82 +50,53 @@ public static void copy(File from, File to, byte[] buf) throws IOException {
     }
 
     /**
-     * Create a copy of an zip file without its empty directories.
+     * Create a copy of a zip file, removing all empty directories.
      * @param inputFile
      * @param outputFile
      * @throws IOException
      */
-    public static void copyZipWithoutEmptyDirectories(final File inputFile, final File outputFile) throws IOException
-    {
+    public static void copyZipWithoutEmptyDirectories(File inputFile, File outputFile) throws IOException {
         final byte[] buf = new byte[0x2000];
+        ZipFile inputZip = new ZipFile(inputFile);
+        ZipOutputStream outputStream = new ZipOutputStream(new FileOutputStream(outputFile));
```

Comment:
Is there a reason 

```
        BufferedOutputStream buffered = new BufferedOutputStream(new FileOutputStream(outputFile));
```

is skipped?

Reply:
It's a mistake. I've put it back. Thanks